docs/contracts.rst

```diff
@@ -249,8 +254,8 @@ inheritable properties of contracts and may be overridden by derived contracts.
 
 ::
 
-    contract owned {
```

Comment:
This might go against the coding style, but `owned` is a property (similar to `mortal`) and thus it makes sense to make it lowercase.